Sources/ArgumentParser/Usage/MessageInfo.swift

```diff
@@ -110,6 +121,8 @@ enum MessageInfo {
         self = .other(message: error.localizedDescription, exitCode: Int32(error.errorCode))
       case let error as LocalizedError where error.errorDescription != nil:
         self = .other(message: error.errorDescription!, exitCode: EXIT_FAILURE)
+      case let error as NSError:
+        self = .other(message: error.localizedDescription, exitCode: Int32(error.code))
```

Comment:
Please do not use the error code as the exit code; see #269. For now, `EXIT_FAILURE` is the best option.

Reply:
@lorentey roger that - done on 37fb5d99ecc3e7d0fa12937514616bd3389caa6e